test/unit/org/authInfo.test.ts

```diff
@@ -1119,15 +1130,18 @@ describe('AuthInfo', () => {
       expect(context.initAuthOptions.called, 'Should have called AuthInfo.initAuthOptions() during refreshFn()').to.be
         .true;
       const expectedInitArgs = {
-        loginUrl: context.getFields().loginUrl,
-        clientId: context.getFields().clientId,
-        privateKey: context.getFields().privateKey,
+        loginUrl: testOrg.loginUrl,
+        clientId: testOrg.clientId,
+        privateKey: testOrg.privateKey,
         accessToken: testOrg.accessToken,
```

Comment:
removed these `context.getFields()` calls to simplify the stubbing (see the new stub only cares about the first and second call)